tests/unit/src/test/resources/semanticTokens3/example/PatternMatching.scala

```diff
@@ -1,23 +1,23 @@
 <<package>>/*keyword*/ <<example>>/*namespace*/
 
 <<class>>/*keyword*/ <<PatternMatching>>/*class*/ {
-  <<val>>/*keyword*/ <<some>>/*variable,readonly*/ = <<Some>>/*class*/(<<1>>/*number*/)
+  <<val>>/*keyword*/ <<some>>/*variable,definition,readonly*/ = <<Some>>/*class*/(<<1>>/*number*/)
   <<some>>/*variable,readonly*/ <<match>>/*keyword*/ {
-    <<case>>/*keyword*/ <<Some>>/*class*/(<<number>>/*variable,readonly*/) <<=>>>/*operator*/
+    <<case>>/*keyword*/ <<Some>>/*class*/(<<number>>/*variable,declaration,readonly*/) <<=>>>/*operator*/
       <<number>>/*variable,readonly*/
   }
 
   <<// tuple deconstruction>>/*comment*/
-  <<val>>/*keyword*/ (<<left>>/*variable,readonly*/, <<right>>/*variable,readonly*/) = (<<1>>/*number*/, <<2>>/*number*/)
+  <<val>>/*keyword*/ (<<left>>/*variable,definition,readonly*/, <<right>>/*variable,definition,readonly*/) = (<<1>>/*number*/, <<2>>/*number*/)
   (<<left>>/*variable,readonly*/, <<right>>/*variable,readonly*/)
 
   <<// val deconstruction>>/*comment*/
-  <<val>>/*keyword*/ <<Some>>/*class*/(<<number1>>/*variable,readonly*/) =
+  <<val>>/*keyword*/ <<Some>>/*class*/(<<number1>>/*variable,definition,readonly*/) =
```

Comment:
I think this is fine since the value is assigned at the start. It's different than pattern matching.

Reply:
What about `number2` ?

Reply:
Ok, the best I can do is make all pattern match definition since everything else will be a bit hacky and it's not really super clear.

Let's say it's a definition because the value is assigned a the same time it's created. This might be different if we have a parial function, but this might hard to make it ideal.